### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by daniduong**

### CODE REVIEW

File: client/src/components/LocationMap.tsx [hunk 1]
```diff
@@ -268,9 +256,8 @@ export function EventsMapView(props: { posts: PostUserPreview[] }) {
       <Paper
         elevation={5}
         style={{
-          height: mapHeight,
-          width: mapWidth,
-          justifyContent: 'center',
+          height: '56vw',
```
Comment (plausibly) [inline client/src/components/LocationMap.tsx:259]: screenshot? how does it look now

**Review by plausibly — APPROVED** [review]

File: client/src/components/LocationMap.tsx (hunk 1 above)
Comment (danielduong8) [reply client/src/components/LocationMap.tsx:259]: test resizing the window and different devices on university-board, it is currently deployed

Comment (plausibly) [reply client/src/components/LocationMap.tsx:259]: it is clipping on my screen

Comment (plausibly) [inline client/src/components/LocationMap.tsx:259]: undo this change, it seems to break on certain screens

**Review by plausibly — CHANGES_REQUESTED** [review]

**Review by plausibly — APPROVED** [review]
